### ISSUE TITLE
Monotype Corsiva Turns into Comic Sans

### ISSUE THREAD

**Issue by jkl-water** [arcusmaximus/YTSubConverter#92]
When I attempt to assign a subtitle the Monotype Corsiva font (it shows up correctly in Aegisub), whether it be through the \fn tag or in the style itself, it appears as Comic Sans after converting to .ytt?

![image](https://github.com/arcusmaximus/YTSubConverter/assets/98797920/33b275c9-c061-43be-a809-abd7b02b8dba)
On Aegisub,

![image](https://github.com/arcusmaximus/YTSubConverter/assets/98797920/824a4678-9018-421b-a36f-ee31f3791533)
and on the youtube video. I hope there's a solution to this!

**Comment by arcusmaximus** [comment]
YouTube subtitles don't actually let you specify a specific font, but only a certain "font family" - a collection of similar-looking fonts. When you select Monotype Corsiva in Aegisub, YTSubConverter will write out the family containing that font, namely "Cursive". For whatever reason, however, YouTube's "Cursive" font family also contains Comic Sans, so that's what you get if your browser thinks Monotype Corsiva is not available.

In short, it's known that this can happen, and I'm afraid there's not really anything we can do about it.

**Comment by jkl-water** [comment]
Ahh, thanks regardless! So there's no way to trick Youtube into using monotype using an .ass file, then, that's a bummer!
